UnityClient/Assets/Scripts/Utils/Extensions/RawImageExtensions.cs

```diff
@@ -0,0 +1,8 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+public static class RawImageExtenions {
+    public static RawImage FindOnCanvas(this RawImage ri) {
+        return (GameObject.Find("Screen") ?? GameObject.Find("Canvas")).GetComponent<RawImage>();
+    }
```

Comment:
My intention with the last comment was to get rid of this `Find` by the objects name. Instead of finding a RawImage, use the editor to set it explicitly and the use the utility class in a extension form... something like this:

```cs
public static RawImage SetLoginBackground(this RawImage image) {
        Texture2D texture = FileManager.Load($"{DBManager.INTERFACE_PATH}bgi_temp.bmp") as Texture2D;
        image.texture = texture;
}
```

Reply:
Oh, I get it! I did the changes following your example